node/core/approval-voting/src/lib.rs

```diff
@@ -1131,7 +1148,18 @@ fn process_wakeup(
 	state: &State<impl DBReader>,
 	relay_block: Hash,
 	candidate_hash: CandidateHash,
+	expected_tick: Tick,
 ) -> SubsystemResult<Vec<Action>> {
+	let mut span = polkadot_node_jaeger::descriptor_span(
```

Comment:
Wouldn't a `hash_span` work as well? Or is the intention to create a different descriptor than what `hash_span` create intentionally?

Reply:
It wouldn't work , I assume, because it wouldn't be unique.